### PR TITLE
Fix MarineVertInterp and update some modules to sky4

### PR DESCRIPTION
**Pull request by danholdaway**
## Description

- Keep up with JEDI (https://github.com/JCSDA-internal/soca/pull/874)
- Keep up with Skylab (to version 4.0)

## Dependencies

N/A

## Impact

N/A

### CODE REVIEW

**Review by jardizzo — APPROVED** [review]

File: src/swell/deployment/platforms/nccs_discover/modules [hunk 1]
```diff
@@ -42,7 +43,7 @@ module use -a /discover/nobackup/drholdaw/JediOpt/modulefiles/core
 module load solo/swell-1.0.0
 module load r2d2/swell-1.0.5
 module load eva/1.3.4
```
Comment (Dooruk) [inline src/swell/deployment/platforms/nccs_discover/modules]: Eva is giving the following error:

`ABORT EvaFactory: Expecting to find a class called 'FigureDriver' in a file called '/discover/nobackup/dardag/JediOpt/src/eva/latest/src/eva/plot_tools/figure_driver.py' but no such file was found or an error occurred during import. `

`Reported error: No module named 'scipy.stats._hypotests_pythran'`

Same issue with EVA 1.3.5 as well.

**Review by Dooruk — CHANGES_REQUESTED** [review]
Hofx and 3dvar worked on my end. I just had to make a new build with jedi_bundle 1.0.10. I will leave the Cylc test to you since you are already in queue.

File: src/swell/deployment/platforms/nccs_discover/modules (hunk 1 above)
Comment (danholdaway) [reply src/swell/deployment/platforms/nccs_discover/modules]: Thanks @Dooruk. I reinstalled `eva/1.3.5` using skylab 4 modules and that should hopefully fix that issue.

Comment (danholdaway) [reply src/swell/deployment/platforms/nccs_discover/modules]: ...and also updated this branch to point to that version of eva.

**Comment by danholdaway** [comment]
Since the nightly is already broken I'm going to merge this so we can see if it works tonight.

**Pull request merged**